
---
float32 x_nose
float32 y_nose
